client/galaxy/scripts/layout/menu.js

```diff
@@ -1,396 +1,417 @@
+            this.$toggle.html( this.model.get( 'title' ) || '' )
+                        .removeClass().addClass( 'dropdown-toggle' )
+                        .addClass( this.model.get( 'cls' ) )
+                        .addClass( this.model.get( 'icon' ) && 'dropdown-icon fa ' + this.model.get( 'icon' ) )
+                        .addClass( this.model.get( 'toggle' ) && 'toggle' )
+                        .attr( 'target', this.model.get( 'target' ) )
+                        .attr( 'href', this.model.get( 'url' ) )
+                        .attr( 'title', this.model.get( 'tooltip' ) )
+                        .tooltip( 'destroy' );
+            this.model.get( 'tooltip' ) && this.$toggle.tooltip( { placement: 'bottom' } );
+            this.$dropdown.removeClass().addClass( 'dropdown' )
+                          .addClass( this.model.get( 'disabled' ) && 'disabled' )
+                          .addClass( this.model.get( 'active' ) && 'active' );
+            if ( this.model.get( 'menu' ) && this.model.get( 'show_menu' ) ) {
+                this.$menu.show();
+                $( '#dd-helper' ).show().off().on( 'click',  function() {
+                    $( '#dd-helper' ).hide();
+                    self.model.set( 'show_menu', false );
+                });
+            } else {
+                self.$menu.hide();
                 $( '#dd-helper' ).hide();
-                self.model.set( 'show_menu', false );
-            });
-        } else {
-            self.$menu.hide();
-            $( '#dd-helper' ).hide();
-        }
-        this.$menu.empty().removeClass( 'dropdown-menu' );
-        if ( this.model.get( 'menu' ) ) {
-            _.each( this.model.get( 'menu' ), function( menuItem ) {
-                self.$menu.append( self._buildMenuItem( menuItem ) );
-                menuItem.divider && self.$menu.append( $( '<li/>' ).addClass( 'divider' ) );
-            });
-            self.$menu.addClass( 'dropdown-menu' );
-            self.$toggle.append( $( '<b/>' ).addClass( 'caret' ) );
-        }
-        return this;
-    },
+            }
+            this.$menu.empty().removeClass( 'dropdown-menu' );
+            if ( this.model.get( 'menu' ) ) {
+                _.each( this.model.get( 'menu' ), function( menuItem ) {
+                    self.$menu.append( self._buildMenuItem( menuItem ) );
+                    menuItem.divider && self.$menu.append( $( '<li/>' ).addClass( 'divider' ) );
+                });
+                self.$menu.addClass( 'dropdown-menu' );
+                self.$toggle.append( $( '<b/>' ).addClass( 'caret' ) );
+            }
+            return this;
+        },
 
-    /** Add new menu item */
-    _buildMenuItem: function ( options ) {
-        var self = this;
-        options = _.defaults( options || {}, {
-            title           : '',
-            url             : '',
-            target          : '_parent',
-            noscratchbook   : false
```

Comment:
I see, it was remedied in https://github.com/galaxyproject/galaxy/pull/3102
Thanks!